### DISCUSSION TITLE
Snacks.nvim Update, Toggle, and Lazygit

### DISCUSSION THREAD

**Discussion by Kayzels** [LazyVim/LazyVim#4728] [Q&A] [answered]
I see a lot of things are moving to snacks.nvim, which is cool, and I like that they're now no longer just in LazyVim.

In my config, I have a custom set up for setting the colorscheme, which I so using `<leader>ub` and I overrode the setup that LazyVim has for that being the background option in Neovim. With toggle, is something like this still possible -- to override the LazyVim toggle and put my own?

Similarly, I disabled LazyVim's lazygit theme integration (set `vim.g.lazygit_config` to false), because I found it led to some issues with colorscheme switching. Would it be possible to use the lazygit integration from snacks without having it also change the theme?

**Comment by folke** [comment, marked as answer]
Yes to both questions. Just check the snacks docs

**Reply by Kayzels** [reply]
Sorry to come back to this, but I'm having an issue with disabling the lazygit theme overriding.

Currently I have
```lua
{
    "folke/snacks.nvim",
    opts = {
      lazygit = {
        configure = false,
        theme = nil,
      },
    },
  }
```

With this, snacks.nvim is still manipulating the LG_CONFIG_FILE env variable, which is what I would like to avoid.

To get an idea of why, I have a Powershell command that I call from Neovim that changes the color scheme for various apps including Lazygit. For Lazygit, it does this by resetting the env var, adding the default config, and then appending the details for the specific colors. So, similar to what the lazygit feature does. The issue with having both do it is that Snacks will append to the variable for the scheme when it initialises. And then when I call the command to update it, it will append to the env var, but it still sees the Snacks config first, so it won't get the new colors.

**Reply by Kayzels** [reply]
Never mind -- was user error, I wasn't correctly setting the environment variable
